crates/re_query/tests/archetype_query_tests.rs

```diff
@@ -41,7 +41,7 @@ fn simple_query() {
 
     // We expect this to generate the following `DataFrame`
     // ┌──────────┬───────────┬────────────┐
-    // │ instance ┆ point2d   ┆ colorrgba  │
+    // │ instance ┆ positions2D┆ colorrgba  │
     // │ ---      ┆ ---       ┆ ---        │
     // │ u64      ┆ struct[2] ┆ u32        │
     // ╞══════════╪═══════════╪════════════╡
```

Comment:
😬

Reply:
these tables don't align properly with my font making fixing them hard >.<

Reply:
Use a better font 🙊

Reply:
but there is no better font than CodeNewRoman!